sshd-osgi/pom.xml

```diff
@@ -84,7 +84,7 @@
                 <executions>
                     <execution>
                         <id>unpack-dependencies</id>
-                        <phase>compile</phase>
+                        <phase>package</phase>
```

Comment:
Have you tested it and made 100% sure that it works as before ? I seem to remember wanting to do this, but something was wrong and only the `compile` phase fixed it. If it is not absolutely necessary for `jacoco` I would rather not change it, otherwise please make 100% sure that the normal build cycle (i.e., `mvn clean install`) creates a viable OSGi artifact (as it currently does).